filepathfilter/filepathfilter.go

```diff
@@ -61,7 +78,7 @@ func (f *Filter) Allows(filename string) bool {
 		}
 	}
 
-	if !included && len(f.include) > 0 {
+	if !included && (len(f.include) > 0 || !f.defaultValue) {
 		tracerx.Printf("filepathfilter: rejecting %q via %v", filename, f.include)
```

Comment:
If `len(f.include) == 0` but `f.defaultValue` is `false`, then we'll return `false` here, which is intended, obviously, but the trace message may be confusing as `f.include` will be empty.  I might recommend adding some additional logic here to output a more meaningful trace message when we're rejecting the match because we fell through to the filter's default value.